### ISSUE TITLE
Error after upgrading from 1.2.0 to 1.2.1

### ISSUE THREAD

**Issue by mtkcs** [serverless-stack/open-next#109]
Hello,

We encoutered this error after upgrading from the version `1.2.0` to `1.2.1`

Here are the logs of the lambda

```
2023-05-10T09:46:48.327Z	d577c19a-2182-4b2e-ad9b-cefe19179b13	ERROR	TypeError: Cannot read properties of null (reading 'useEffect')
    at exports.useEffect (/var/task/node_modules/next/dist/compiled/react/cjs/react.production.min.js:28:292)
    at MyApp (/var/task/.next/server/pages/_app.js:130:34)
```

It seems that it is related to `__NEXT_PRIVATE_PREBUNDLED_REACT`.  Any idea how would we fix the error, we're using `Next.js@13.4.1`?

**Comment by dreamorosi** [comment]
I'm having the same error

**Comment by fwang** [comment]
Hey @mtkcs @dreamorosi can you post in #open-next on Discord — http://discord.gg/sst

A couple of experts in the community are more familiar with this flag.

**Comment by dreamorosi** [comment]
Thanks @fwang, already did that - it looks that it might be due to an upstream (Next.js) issue: https://discord.com/channels/983865673656705025/1105529905892036778/1105868486955257887

**Comment by fwang** [comment]
Fixed by @conico974 in v1.3.2
